File: PYTHON/Program/ParkingLot.py
class ParkingLot:
    __Kapasitas = 0
    __JumlahNow = 0
    
    def __init__(self, Kapasitas, JumlahNow):
        self.Kapasitas = Kapasitas
        self.JumlahNow = JumlahNow
        
    def get_Kapasitas(self):
        return self.Kapasitas

    def set_Kapasitas(self, Kapasitas):
        self.Kapasitas = Kapasitas
        
    def get_JumlahNow(self):
        return self.JumlahNow

    def set_JumlahNow(self, JumlahNow):
        self.JumlahNow = JumlahNow
        
    def TampilkanParkingLot(self):
        print("Parking Lot")
        print(f"Kapasitas       : {self.Kapasitas} Kendaraan")
        print(f"Jumlah Sekarang : {self.JumlahNow} Kendaraan")
    

File: CPP/Program/ParkingLot.cpp
#include <iostream>

#pragma once

class ParkingLot {
private:
    int Kapasitas;
    int JumlahNow;

public:
    ParkingLot(int Kapasitas, int JumlahNow) : Kapasitas(Kapasitas), JumlahNow(JumlahNow) {}

    int get_Kapasitas() const {
        return Kapasitas;
    }

    void set_Kapasitas(int Kapasitas) {
        this->Kapasitas = Kapasitas;
    }

    int get_JumlahNow() const {
        return JumlahNow;
    }

    void set_JumlahNow(int JumlahNow) {
        this->JumlahNow = JumlahNow;
    }

    void TampilkanParkingLot() const {
        std::cout << "Parking Lot" << std::endl;
        std::cout << "Kapasitas       : " << Kapasitas << " Kendaraan" << std::endl;
        std::cout << "Jumlah Sekarang : " << JumlahNow << " Kendaraan" << std::endl;
    }
};